.ci/windows/windows_proj_jjb.yaml

```diff
@@ -0,0 +1,77 @@
+- job-template:
+    name: "{jjb_proj}"
+    project-type: freestyle
+    node: r-aa-fatty27
+    folder: Sockperf
```

Comment:
all other folders are lowercase, so this one should be called sockperf instead of Sockperf